op-node/rollup/chain_spec.go

```diff
@@ -123,6 +123,11 @@ func (s *ChainSpec) IsIsthmus(t uint64) bool {
 	return s.config.IsIsthmus(t)
 }
 
+// IsJovian returns true if t >= jovian_time
+func (s *ChainSpec) IsJovian(t uint64) bool {
```

Comment:
where to use `IsJovian` vs `IsConfigurableMinBaseFee`?

Reply:
I think for this HF it is ok to just use `IsJovian`